src/View/Helper/PaginatorHelper.php

```diff
@@ -820,9 +822,10 @@ protected function _firstNumber($ellipsis, $params, $start, $options)
     protected function _lastNumber($ellipsis, $params, $end, $options)
     {
         $out = '';
+        $last = is_int($options['last']) ? $options['last'] : 0;
```

Comment:
No, with `is_numeric()` it does the same as before :
```php
var_dump(is_numeric('zz')); // false
var_dump(is_numeric('1')); // true
var_dump(is_numeric(1));  // true

var_dump(is_int('zz')); // false
var_dump(is_int('1')); // false
var_dump(is_numeric(1));  // true
```


The expected behavior, is to detect `'1'`, `'2'`, ... ` '9'` as a string and not as an integer.